docs/event-bus/docs/013-details-service-programming-model.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Service Programming Model
+type: Details
+---
+# Service Programming Model
```

Comment:
this header is going to be duplicated in the UI, because the title handles that as well, so please remove it

Reply:
Will remove it